MimeKit/Encodings/Base64Decoder.cs

```diff
@@ -37,7 +37,7 @@ namespace MimeKit.Encodings {
 	/// </remarks>
 	public class Base64Decoder : IMimeDecoder
 	{
-		static readonly byte[] base64_rank = new byte[256] {
+		static ReadOnlySpan<byte> base64_rank => new byte[256] {
 			255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,
```

Comment:
should this be `static readonly ReadOnlySpan<byte> ...`? and why `=>` vs `=`?

Reply:
Agree -- this is a really weird syntax that requires you to understand how Roslyn transforms it. It looks like it's allocating on every use, but Roslyn burns the data directly into the data file instead. This allows the code to access the data directly and eliminate various indirections and bounds checks.